<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wavefrontFilterDlg</class>
 <widget class="QDialog" name="wavefrontFilterDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>327</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Wavefronts filter</string>
  </property>
  <property name="whatsThis">
   <string>You can set a RMS threshold that will delete any loaded wavefronts and optionally any
 saved wavefronts that have RMS values over the threshold.

You can hover the mouse over a dot on the plots to see the wavefront name. 

You can click on the dot and it will be set as the current displayed wavefront.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filter if </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RMS &gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="rmsValue">
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="remove">
       <property name="toolTip">
        <string>Do not add wavefront to list if RMS is too big.</string>
       </property>
       <property name="text">
        <string>Skip file if RMS is too big.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="deleteFile">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Delete wavefront file from file system.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="beepEnable">
       <property name="toolTip">
        <string>Enable beep in batch interferogram processing if RMS is too big.</string>
       </property>
       <property name="text">
        <string>Beeb if RMS is greater</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
     <item>
      <widget class="astigScatterPlot" name="astigPlot" native="true"/>
     </item>
     <item>
      <widget class="rmsPlot" name="rmSPlot" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>astigScatterPlot</class>
   <extends>QWidget</extends>
   <header>astigscatterplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>rmsPlot</class>
   <extends>QWidget</extends>
   <header>rmsplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>wavefrontFilterDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>wavefrontFilterDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
